docs/projects/capitalize/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Capitalize in Every Language
+layout: default
+date: 2019-03-31
+last-modified: 2019-03-31
+featured-image:
+tags: [capitalize]
+authors:
```

Comment:
You can add yourself here! We have an authors doc: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml. Put whatever you want there and then place your username here like:

```yml
authors:
  - your_username
```